.pre-commit-config.yaml

```
@@ -13,8 +13,8 @@ repos:
   - id: codespell
     args: ["--ignore-words", "doc/styles/config/vocabularies/ANSYS/accept.txt", "-w"]
 
-- repo: https://github.com/pre-commit/mirrors-prettier
-  rev: 'v4.0.0-alpha.8'
+- repo: "https://github.com/pycontribs/mirrors-prettier"
```

Why are the quotes needed?
```suggestion
- repo: https://github.com/pycontribs/mirrors-prettier
```